cmd/manager/main.go

```diff
@@ -72,7 +72,7 @@ func main() {
 	// Create a new Cmd to provide shared dependencies and start components
 	mgr, err := manager.New(cfg, manager.Options{
 		Namespace:          namespace,
-		MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
+		MetricsBindAddress: fmt.Sprintf("%s:%d", operatorSdkMetricsHost, operatorSdkHttpMetricsPort),
```

Comment:
I'm not certain as to if I can/should use the same port as operatorSDK (8383), 
when I tried it failed on "address already set".
if so then I'm not sure if the which port value does `MetricsBindAddress` should contain

Reply:
I ended up removing the operator-sdk metric altogether..  the way I see it, if we don't use them,  we shouldn't keep them

Reply:
update - brought it back :)